components/docs-chef-io/content/automate/ha_cert_selfsign.md

```diff
@@ -75,3 +75,8 @@ You can create a self-signed key and certificate pair with the **OpenSSL** utili
     ```
 
 1. The script generates the certificates at the newly created directory, `rotate-certs` in this case.
+
+{{< note >}}
+Please refer Opensearch certificate [documentation](https://opensearch.org/docs/1.2/security-plugin/configuration/tls/#x509-pem-certificates-and-pkcs-8-keys)
```

Comment:
Any specific reason for mentioning this note? If so we may need to provide some context in the note